gems/sorcery/0.16/sorcery.rbs

```diff
@@ -0,0 +1,22 @@
+module Sorcery
+  module Model
+    def authenticates_with_sorcery!: () -> void
+
+    module ClassMethods[T]
+      def authenticate: (*String credentials) ?{ () -> untyped } -> (T | nil)
+    end
+  end
+
+  module Controller
+    module InstanceMethods[T]
+      def require_login: () -> (nil | void)
+      def login: (*String credentials) ?{ (T, nil) -> untyped } -> (T | nil | untyped)
+
+      def logged_in?: () -> bool
+      def current_user: () -> (T | nil)
+      def logout: () -> (nil | void)
+      def auto_login: (T, ?bool) -> T
```

Comment:
How about adding a parameter name? Especially, users cannot imagine the second parameter's meaning from `?bool`.